tasks/main.yml

```diff
@@ -2,8 +2,7 @@
 - name: Include opensearch installation
   ansible.builtin.import_tasks: opensearch.yml
 
-- name: Start and enable OpenSearch service
+- name: Enable OpenSearch service
   ansible.builtin.systemd:
     name: opensearch
     enabled: true
-    state: started
```

Comment:
Removing this means that if I go to my OpenSearch server run `systemctl stop opensearch` and later run this role to ensure everything is set up correctly, OpenSearch will not be started. That probably shouldn't be.